.circleci/config.yml

```diff
@@ -22,11 +22,6 @@ e2e-kind-test: &e2e-kind-test
     - setup_remote_docker
     - checkout:
         path: /go/src/github.com/cisco-app-networking/nsm-nse
-    - run:
```

Comment:
There's a lot of functional changes in here that don't have anything to do with renaming.  It's a bit much to shove into the single renaming PR.  Please propose the functional changes as a different PR.